### PR TITLE
Add PySpark examples and separate tabs in Unity Catalog Apache Spark …

### LINKED ISSUE #548: [DOCS] Breakout code blocks in Spark page
**Is your feature request related to a problem? Please describe.**

Let's clean up some of code blocks on the [Apache Spark](https://docs.unitycatalog.io/integrations/unity-catalog-spark/) docs.

* The *Merge mytable with srctable* should be broken out into separate commands with title.

* The output should be in `console` mode while the code should be in `sql` or `python` depending on the language used.  

* We should have `Spark SQL` and `PySpark` commands (currently its only showing Spark SQL) for the *Running CRUD Operations on a Unity Catalog table* section.

 

**Describe the solution you would like**

**Additional context**

### PR DESCRIPTION
**Pull request by vksx**
…integration document.

**PR Checklist**

- [x] A description of the changes is added to the description of this PR.
- [x] If there is a related issue, make sure it is linked to this PR.
- [x] If you've fixed a bug or added code that should be tested, add tests!
- [x] If you've added or modified a feature, documentation in `docs` is updated

**Description of changes**

Added PySpark examples and separate tabs to the Unity Catalog Apache Spark integration documentation, including breaking out the merge command.

Closes #548

### CODE REVIEW

**Comment by dennyglee** [comment]
Thanks @vksx - could you also share a deployed GitHub pages of these docs so to make it easier to review visually?  Thanks!

File: docs/integrations/unity-catalog-spark.md [hunk 1]
```diff
@@ -229,7 +229,7 @@ Let’s query the first five rows of the `marksheet` table.
 === "PySpark"
 
     ```python
-    sql("SELECT * FROM default.marksheet LIMIT5;").show()
+    sql("SELECT * FROM default.marksheet LIMIT 5").show()
```
Comment (dennyglee) [inline docs/integrations/unity-catalog-spark.md:232]: Thanks for fixing the spacing, could you add a semi-colon to the end of the statement?  Thanks!

**Review by dennyglee — CHANGES_REQUESTED** [review]

File: docs/integrations/unity-catalog-spark.md (hunk 1 above)
Comment (vksx) [reply docs/integrations/unity-catalog-spark.md]: Added.

**Comment by vksx** [comment]
Thanks @dennyglee for reviewing this! You can see the changes here: https://vksx.github.io/unitycatalog/integrations/unity-catalog-spark/

**Review by dennyglee — CHANGES_REQUESTED** [review]
Minor nit - in the early `PySpark` code examples, we use `sql("$SQL statement$")` and in later sections we use `spark.sql("$SQL statement$)`.  Perhaps update the earlier statements to match the later ones?  Thanks!

**Review by dennyglee — APPROVED** [review]
Thanks, LGTM!

**Pull request merged**